client/components/SocialClub/SocialClubPage.js

```diff
@@ -45,20 +40,19 @@ class SocialClubPage extends Component {
 
 
   componentDidMount() {
-    this.props.getClub(3);
-    toastr.success('Clubs loaded Successfully');
+    window.scrollTo(0, 0);
```

Comment:
Oh! Is this to ensure on page load the person is at the top of the page?
If we remove this, does the user land in the middle of the page? Because I believe if we can do without it, then there's no point in having it.

Reply:
Surprisingly, it does land the user in the middle section of the page.